src/D2L.Bmx/D2L.Bmx.csproj

```diff
@@ -1,31 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
```

Comment:
is this an auto-formatting thing or are you explicitly removing these blank lines?
I'd prefer keeping them since this is how the default csproj files from templates look like.

Reply:
I think it was my visual studio doing that. It also flipped that `+` to `-` issue we had before but then I resaved in vscode. I'll add back in the lines it removed